interop/script/run.exs

```diff
@@ -13,49 +13,46 @@ Logger.configure(level: level)
 Logger.info("Rounds: #{rounds}; concurrency: #{concurrency}; port: #{port}")
 
 alias Interop.Client
+alias GRPC.Client.Adapters.{Mint, Gun}
```

Comment:
```suggestion
alias GRPC.Client.Adapters.Mint
alias GRPC.Client.Adapters.Gun
```

I prefer this style because it makes for easier refactoring and find-all

Reply:
That makes sense. Tks